classes/Profile.php

```diff
@@ -128,7 +128,7 @@ public static function getProfileAccess($idProfile, $idTab)
      * @param int $idProfile Profile ID
      * @param string $type Type
      *
-     * @return bool
+     * @return bool|array
```

Comment:
Would be nice to define the array structure. I like Psalm's [object-like array definitions](https://psalm.dev/docs/annotating_code/type_syntax/array_types/#object-like-arrays).

Otherwise this can be specified like this:
```suggestion
     * @return array|false
```